presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -41,6 +45,20 @@ protected boolean isParameterizedVarcharSupported()
         return true;
     }
 
+    protected boolean canCreateSchema()
+    {
+        return true;
+    }
+
+    protected boolean canDropSchema()
+    {
+        return true;
+    }
+
+    protected void cleanUpSchemas(List<String> schemaNames)
+            throws SQLException
+    {}
```

Comment:
throw an exception when this method is not implemented. So in case someone return `false` from `canDropSchema` we don't leave a trash behind.